### DISCUSSION TITLE
Пропало быстрое окошко с удалением и тайм-аутом сообщения \ The quick window with deletion and message timeout disappeared

### DISCUSSION THREAD

**Discussion by fakenetglowboiiiTV** [obsproject/obs-studio#11463] [Development Questions]
Мужики-женики, пропало быстрое окошко с удалением и тайм-аутом сообщения, не могу понять почему и как восстановить, всё облазил. Типо, зритель пишет, на него пкм и было очень удобное окно быстрого реагирования, сейчас его нет, вместо этого "изучить, скопировать, приблизить", вообще не то, бред какой-то. Как починить и вернуть?

Guys, the quick window with deleting and timeout messages has disappeared, I can't understand why and how to restore it, I've looked everywhere. Like, the viewer writes, right-click on him and there was a very convenient quick response window, now it's gone, instead of it there is "study, copy, zoom", not at all, some kind of nonsense. How to fix and return?

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
